src/main/java/org/broadleafcommerce/frameworkmapping/FrameworkMappingHandlerMapping.java

```diff
@@ -98,7 +98,9 @@ protected boolean isHandler(Class<?> beanType) {
 
     @Override
     protected RequestMappingInfo getMappingForMethod(Method method, Class<?> handlerType) {
-
+        // New approach is to redirect instead of matching trailing slash.
+        // However, this can have performance implications. Keeping deprecated approach for now.
+        getBuilderConfiguration().getPatternParser().setMatchOptionalTrailingSeparator(true);
```

Comment:
Interesting. I [found this SO post](https://stackoverflow.com/a/74786037) containing further links and context about why they made this change.

I guess for now the easy approach is to stick to the deprecation, and in the future when we _must_ have strict matching, we will likely need to update the gateways to always rewrite paths with/without trailing slash. 

I'm curious though - it looks like the builder configuration is fairly constant - can we move this to the `afterPropertiesSet` method so it's called fewer times?
```java
    @Override
    public void afterPropertiesSet() {
        super.afterPropertiesSet();
        // New approach is to redirect instead of matching trailing slash.
        // However, this can have performance implications. Keeping deprecated approach for now.
        getBuilderConfiguration().getPatternParser().setMatchOptionalTrailingSeparator(true);
    }
```